packages/apiaryb-parser/package.json

```diff
@@ -26,10 +26,10 @@
     "deckardcain": "^1.0.0"
   },
   "peerDependencies": {
-    "@apielements/core": ">=0.1.0 <0.3.0"
+    "@apielements/core": ">=0.1.0 <0.4.0"
```

Comment:
As this adapters and others won't get release here, the current versions won't be compatible with the new core. I think we either need to release all the adapters, or make the change compatible with the old range.

Given the API has changed and now the `serialiseSync` is implemented in a way it will not break with old adapters (https://github.com/apiaryio/api-elements.js/pull/549#discussion_r478455762) then possibly going back to releasing core as `0.2.1` might make sense.

Reply:
Reverted the version bump in 877bfb4, and added the only missing `serializeSync` in fc81a6d.